dune

```
@@ -2,3 +2,5 @@
  (targets graphql_schema.json)
  (mode promote)
  (action (with-stdout-to graphql_schema.json (run %{exe:src/app/graphql_schema_dump/graphql_schema_dump.exe}))))
+
+(data_only_dirs opam_switches)
```

```suggestion
(dirs :standard \ opam_switches)
```
so this is the suggestion, I'll apply it :)